test/test.js

```diff
@@ -6,6 +6,12 @@ chai.use(dirtyChai)
 const generateIndex = require('../lib/index')
 const mockContentCatalog = require('./mock-content-catalog.js')
 
+const lunr = require('lunr')
+const lunr2 = require('lunr')
```

Comment:
`require('lunr')` will cache the instance, so unless I'm missing something, it's effectively the same as:

```js
const lunr = require('lunr')
const lunr2 = lunr
```

Just did a quick test and the tests suite is working just fine if I remove `lunr2` and use `lunr` instead.